mkdocs_simple_plugin/plugin.py

```diff
@@ -15,7 +15,8 @@ class SimplePlugin(BasePlugin):
     config_scheme = (
         ('include_folders', config_options.Type(list, default=['*'])),
         ('ignore_folders', config_options.Type(list, default=[])),
-        ('ignore_hidden', config_options.Type(bool, default=True))
+        ('ignore_hidden', config_options.Type(bool, default=True)),
+        ('include_extensions', config_options.Type(list, default=['.md']))
```

Comment:
There's no reason to include `.md` here since you're forcing the inclusion below

```suggestion
        ('include_extensions', config_options.Type(list, default=[]))
```